src/containerapp/azext_containerapp/_utils.py

```diff
@@ -361,6 +361,12 @@ def parse_secret_flags(secret_list):
         identity_Id = ""
 
         kv_identity = value.split(',', 2)
+        if len(kv_identity) == 1:
```

Comment:
What if it starts with `keyvaultref` and `identityref` is not present?

Reply:
Next line we will do the check and will return "Identityref is missing. Secrets must be in format \"<key>=<value> <key>=<value> ...\" or \"<key>=<keyvaultref:keyvaulturl,identityref:indentityId> ...\."